### PR TITLE
Improvement: Add Object/Toolpath switch display interaction

### PR DESCRIPTION
**Pull request by bbaa77770**
[LU-714](https://snapmaker2.atlassian.net/browse/LU-714)

### CODE REVIEW

File: src/app/toolpaths/ToolPathGroup.js [hunk 1]
```diff
@@ -50,6 +50,15 @@ class ToolPathGroup {
         this.simulationObjects.visible = false;
     }
 
+    showToolpathObjects(show) {
+        this.toolPathObjects.visible = show;
```
Comment (parachvte) [inline src/app/toolpaths/ToolPathGroup.js:54]: `this.toolPathObjects` is nullable?

Comment (parachvte) [reply src/app/toolpaths/ToolPathGroup.js:54]: `showToolpathObjects` uses objects
`showSimulationObject` uses object, there are `this.simulationObjects` and `this.simulationObject`, what's the differences?

File: src/app/widgets/CncLaserTopRight/styles.styl [hunk 2]
```diff
@@ -0,0 +1,62 @@
+@import "../styles.styl"
+
+$z-index-float = 500
+$visualizer-padding = 10px
+
+.visualizer-top-right
```
Comment (parachvte) [inline src/app/widgets/CncLaserTopRight/styles.styl:6]: Use { braces

File: src/app/widgets/LaserVisualizer/Visualizer.jsx [hunk 3]
```diff
@@ -292,8 +294,15 @@ class Visualizer extends Component {
                         <VisualizerTopLeft />
                     </div>
                 )}
+                {(!isEditor && displayedType === 'toolpath') && (
```
Comment (parachvte) [inline src/app/widgets/LaserVisualizer/Visualizer.jsx:297]: `DISPLAYED_TYPE_TOOLPATH`

**Review by parachvte — CHANGES_REQUESTED** [review]
Check procedure before merging PR.

**Review by zjiefee — APPROVED** [review]

**Review by parachvte — APPROVED** [review]

**Pull request merged**